README.md

```diff
@@ -41,7 +41,22 @@ in your commit messages so it is clear what each commit is for._" <br />
 when you want someone to review your work..._" <br />
 > "_Please only assign a Pull Request for review
 once all the tests have passed on CI..._" <br />
-> For a **list** of these common scenarios see:
+
+We've devised a set of rules and actions to respond to these problems, they look like this:
+
+| Category      | Rule                                                                               | dwylbot comments                                                                                                                                                                                                                                                                                                      | Labels changed                                                                   |
```

Comment:
I think rules and triggers are slightly different (but sometime they can be the same).
For me the rule is all the conditions the issue/pr must satisfied. The trigger is what is going to start the verification of the rule. Different type of triggers can also start the same rule.

For example for the rule that check if an issue has the description will only be triggered when the **issue is created** and dwylbot shouldn't run the rule on other triggers, ie we don't want the rule when we close the issue, or when someone change a label,...
Let me know what you think, maybe I'm going to much in details for the readme but I think it can help users to reproduce a error if they need to test it.